### PR TITLE
[DESK-803] icons remoteit components

### PR DESCRIPTION
**Pull request by mauricio0000**
### Description
 icons remoteit components

### Checklist

image

### Test plan

 icons slow to load in Admin Panel

### Ticket

https://remoteit.atlassian.net/browse/DESK-803

### CODE REVIEW

**Review by JamieRuderman — CHANGES_REQUESTED** [review]
Looks good, I just don't think we want the changes in these files:
- SearchField.tsx 
- RefreshButton.tsx 
- DeviceSetupItem.tsx 

Also you should see if we can remove the font files.

**Comment by mauricio0000** [comment]
@JamieRuderman 

Font files were deleted 

https://github.com/remoteit/desktop/blob/f854eeba586bdd0655e090ede8a81fb788c15871/frontend/src/components/Icon/vendor/all.min.css

https://github.com/remoteit/desktop/blob/f854eeba586bdd0655e090ede8a81fb788c15871/frontend/src/components/Icon/Icon.css

File: frontend/src/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -19,23 +22,32 @@ export interface IconProps {
...
-    if (color) styles.color = colors[color]
-    if (inline) styles.marginLeft = size ? fontSizes[size] / 2 : spacing.md
-    if (inlineLeft) styles.marginRight = size ? fontSizes[size] / 2 : spacing.md
-    if (size) styles.fontSize = fontSizes[size]
-
-    return <span className={classes} {...props} style={styles} ref={ref} />
+  switch (type) {
+    case 'brands': {
+      fontType = 'fab'
+      break
```
Comment (JamieRuderman) [inline frontend/src/components/Icon/Icon.tsx:48]: We also need to support the `regular` and `solid` weights.

**Review by JamieRuderman — CHANGES_REQUESTED** [review]
Please see comments thanks!

**Review by JamieRuderman — APPROVED** [review]
Thanks!

**Pull request merged**